### PR TITLE
Fix using IKFrame with MoveRelative

### PR DESCRIPTION
**Pull request by v4hn**
`direction` was interpreted relative to the link even when an `ik_frame` was specified.

Solves #318 .

### CODE REVIEW

**Review by rhaschke — APPROVED** [review]
Thanks, Michael. Please have a look at my simplify commit and finally merge if OK.

File: core/test/test_move_relative.cpp [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		twist.header.frame_id = "world";
+		twist.twist.angular.z = TAU / 8.0;
+		return twist;
+	}());
+
+	ASSERT_TRUE(t.plan()) << "Failed to plan";
+
+	const auto& tip_name{ group->getOnlyOneEndEffectorTip()->getName() };
+	auto start_eef_position{ position(scene, tip_name) };
+	auto end_eef_position{ position(move->solutions().front()->end()->scene(), tip_name) };
```
Comment (jliukkonen) [inline core/test/test_move_relative.cpp:97]: Can these variables be marked `const`?

File: core/test/test_move_relative.cpp [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+		geometry_msgs::TwistStamped twist;
+		twist.header.frame_id = "world";
+		twist.twist.angular.z = TAU / 8.0;
+		return twist;
+	}());
+
+	ASSERT_TRUE(t.plan());
+
+	auto start_eef_position{ position(scene, ATTACHED_OBJECT) };
+	auto end_eef_position{ position(move->solutions().front()->end()->scene(), ATTACHED_OBJECT) };
```
Comment (jliukkonen) [inline core/test/test_move_relative.cpp:120]: Can these variables be marked `const`?

**Review by jliukkonen — COMMENTED** [review]
Just couple small nitpicks. Thanks for the fix!

**Review by JafarAbdi — APPROVED** [review]

**Pull request merged**